client/src/app/openslides-main-module/services/app-injector.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injector } from '@angular/core';
+
+export class AppInjector {
+    private static injector: Injector;
+
+    static setInjector(injector: Injector) {
+        AppInjector.injector = injector;
+    }
+
+    static getInjector(): Injector {
+        return AppInjector.injector;
+    }
```

Comment:
In every file, you use the `Injector`, you write something like
```js
    const injector = AppInjector.getInjector();
    const service = injector.get(ServiceToken);
```

Could you provide some methods directly, which invokes the methods provided by the `Injector`?
Something like:

```suggestion
    public static get<Token, Service>(token: Token): Service {
        return AppInjector.injector.get(token);
    }
```

Reply:
Good call, changed it accordingly.